### DISCUSSION TITLE
Proof type is not visible! How to contact with our GitHub?

### DISCUSSION THREAD

**Discussion by luiox** [community/community#66761] [GitHub Education]
### Hi everyone,

I have encountered the same trouble as this https://github.com/orgs/community/discussions/65142.
I spent a long time trying various methods provided below, including changing different browsers (using Chrome, Edge, Firefox), using their private mode, clearing cookies, and so on. But none of them worked.
 I hope to find a solution or tell me how to contact with Github.

**Comment by arafajs** [comment]
https://support.github.com/request

**Reply by luiox** [reply]
What is your mean
